### PR TITLE
Updated the default values of parameter of driver deployed via csm-operator and removed the broken link

### PR DESCRIPTION
**Pull request by chimanjain**
# Description
* Updated the proper default values of parameter of driver deployed via csm-operator.
* Removed the broken link and note associated to it as it is not necessary anymore.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1615 |
| https://github.com/dell/csm/issues/1559 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shefali-malhotra — APPROVED** [review]
LGTM

**Review by panigs7 — APPROVED** [review]

File: content/docs/deployment/csmoperator/drivers/powerflex.md [hunk 1]
```diff
@@ -145,9 +148,10 @@ For detailed PowerFlex installation procedure, see the [Dell PowerFlex Deploymen
 
    | Parameter | Description | Required | Default |
    | --------- | ----------- | -------- |-------- |
+   | namespace | Specifies namespace where the driver will be installed | Yes | "vxflexos" |
    | dnsPolicy | Determines the DNS Policy of the Node service | Yes | ClusterFirstWithHostNet |
-   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType`. In OCP <= 4.16 and K8s <= 1.29, fsGroupPolicy is an immutable field. | No | "ReadWriteOnceWithFSType" |
-   | replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, excess pods will become stay in a pending state. Defaults are 2 which allows for Controller high availability. | Yes | 2 |
+   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType`. In OCP <= 4.16 and K8s <= 1.29, fsGroupPolicy is an immutable field. | No | "File" |
```
Comment (prablr79) [inline content/docs/deployment/csmoperator/drivers/powerflex.md:153]: these changes applicable to previous version of docs ? if yes can you use the same PR for updating previous v1,v2 etc ?

**Review by prablr79 — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csmoperator/drivers/powerflex.md (hunk 1 above)
Comment (chimanjain) [reply content/docs/deployment/csmoperator/drivers/powerflex.md:153]: Yes @prablr79 , great catch, update the parameters for v1, v2, v3 versions also.

Comment (chimanjain) [reply content/docs/deployment/csmoperator/drivers/powerflex.md:153]: Please review

Comment (chimanjain) [reply content/docs/deployment/csmoperator/drivers/powerflex.md:153]: Some fields were missing in the previous versions, updated them also.

**Review by rajkumar-palani — APPROVED** [review]
LGTM

**Review by prablr79 — APPROVED** [review]
LGTM

**Pull request merged**